pkg/minikube/machine/cluster_test.go

```diff
@@ -412,6 +413,8 @@ func TestGetHostStatus(t *testing.T) {
 
 func TestCreateSSHShell(t *testing.T) {
 	api := tests.NewMockAPI(t)
+	// Setting the default client to native gives much better performance.
```

Comment:
Why is performance important, to fixing a unit test stability issue ?

Reply:
I copied the comment from pkg/minikube/cluster.go init func.

https://github.com/kubernetes/minikube/blob/e6ba335e9c5e5d7e37065e00d67cb9340b65a713/pkg/minikube/cluster/cluster.go#L41

There is no other reason.
Should I delete this comment?

Reply:
It is fine if moving to the native client fixes the timing issues with the test, but that is not really "performance" (the test has different requirements). Just wanted the comment to be improved.

Reply:
Thanks. I’ll fix this comment.

Reply:
Updated